src/Indicator.vala

```diff
@@ -541,6 +545,20 @@ public class Sound.Indicator : Wingpanel.Indicator {
 
         return true;
     }
+
+    private void update_tooltip () {
+        debug ("VolumeControl ready? %s".printf (volume_control.ready.to_string ()));
+
+        string description = _("Volume: %.0f%%").printf (
+            (volume_control.mute) ? 0 : volume_control.volume.volume * 100
+        );
+        string accel_label = _("Middle-click to %s").printf (
+            (volume_control.mute) ? _("unmute") : _("mute")
+        );
```

Comment:
```suggestion
        string accel_label = volume_control.mute ? _("Middle-click to unmute") : _("Middle-click to mute");
```

It would be good for DRY to construct similar sentences in this way but I guess it might make translating difficult in some languages.

Reply:
Actually, I'm not sure if it'll make translating more difficult. If anything, it'll be easier since they have a full sentence to translate instead of trying to juggle between words within a sentence. I'll apply your suggestion. Thank you!